tests/peer/meshRoom.js

```diff
@@ -274,6 +274,32 @@ describe('MeshRoom', () => {
         assert.equal(answerSpy.callCount, 1);
         assert(answerSpy.calledWith(meshRoom._localStream));
       });
+
+      it('should not create MediaConnection when has a connection with the peer and self peerId is larger', () => {
+        const testItems = [
+          { src: 'bbbbbbb', self: 'bbbbbbbb' },
```

Comment:
I think the property name `src` does not sufficiently represent what it stands for.
IMO, the `src` property and the `self` property mean the remote peer ID and the local peer ID, respectively.